examples/collaborative_filtering/example2.c

```diff
@@ -50,7 +89,9 @@ int main(int argc, char *argv[]) {
   char *uid = NULL; // user id 
   user *u;
   distance distances[MAXDISTANCES];
-  int total_distances;
+  int total_distances; 
+  rating *results[MAXRECOMMENDATIONS + 1] = {0}; 
```

Comment:
Because the elements are pointers, they will be defaulted to NULL. If type elements where a non-pointer and non-void type they would default to the "0" equivalent.